dpgen/generator/run.py

```diff
@@ -2537,8 +2791,13 @@ def post_fp_vasp (iter_index,
                 if all_sys is None:
                     all_sys = _sys
                 else:
-                    all_sys.append(_sys)
+                    try:
+                        all_sys.append(_sys)
+                    except:
+                        dlog.info('%s has different formula, so pass in line 3518'%oo)
```

Comment:
It is not a good idea to hard coding the line number. 

Please check the solutions : https://stackoverflow.com/questions/3056048/filename-and-line-number-of-python-script